lib/elixir/src/elixir_lexical.erl

```diff
@@ -46,6 +47,9 @@ record_import({Module, Function, Arity}, Ref) ->
 record_remote(Module, Function, Ref) ->
   if_tracker(Ref, fun(Pid) -> ?tracker:remote_dispatch(Pid, Module, mode(Function)), ok end).
 
+record_remote(Module, Function, ReferencedFunction, Arity, Line, Ref) ->
```

Comment:
I know this is a super minor nitpick but can we order the arguments are: `record_remote(Module, Fun, Arity, EnvFunction, Line, Ref)`? I think Fun and Arity belong conceptually closer to Module.